hackathon_site/dashboard/frontend/.yarnrc

```diff
@@ -0,0 +1 @@
+--install.frozen-lockfile true
```

Comment:
does this mean we can just run yarn install from now on instead of the full command or it has nothing to do with our files?

Reply:
That's the hope! As long as you're running yarn v1, this should work (we'll need a new solution for yarn 2)